interview-questions.md

```diff
@@ -9,59 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: I am not entirely sure but I think that "this" is a part of react that can be used in Javascript. I think it may involve using functions. 
 
-  Researched answer:
+  Researched answer: "This" is a key word in javascript that refers to an object. It is used in a function to show where to get value or data from. 
```

Comment:
Good research. The syllabus gives good examples. 
https://github.com/learn-academy-2022-bravo/Syllabus/blob/main/javascript/classes.md